packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -64,9 +65,30 @@ const entryPointExists = (segments) => {
     return false
 }
 
-const findInProjectThenSDK = (pkg) => {
+/**
+ * Find the path to a dependency
+ * First check the project's node_modules, then the SDK's node_modules, then the monorepo's node_modules
+ * @param {string} pkg - name of the dependency
+ * @returns {string} path to the dependency
+ */
+const findDependency = (pkg) => {
     const projectPath = resolve(projectDir, 'node_modules', pkg)
```

Comment:
I think to be aligned with the original implementation (find in project first and then the sdk), I would check `isMonoRepo` earlier in the code block.

```suggestion
    const projectPath = isMonoRepo ? resolve(projectDir, '..', '..', 'node_modules', pkg) : resolve(projectDir, 'node_modules', pkg)
```